test/room_test.rb

```diff
@@ -9,16 +9,20 @@ def test_it_exists
   end
 
   def test_it_has_a_category
-    skip
     room = Room.new(:bedroom, 10, '13')
     assert_equal :bedroom, room.category
   end
 
   def test_it_can_get_area
-    skip
     room1 = Room.new(:bedroom, 10, '13')
     room2 = Room.new(:living_room, 15, '12')
     assert_equal 130, room1.area
     assert_equal 180, room2.area
   end
+
+  def test_it_has_paint
```

Comment:
Great job on this test!

Reply:
@memcmahon Thank you for the feedback! I'll work with my mentors to get it done :)